sys-cluster/charliecloud/charliecloud-0.15.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit autotools python-single-r1
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/hpc/${PN}.git"
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/hpc/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="Lightweight user-defined software stacks for high-performance computing"
+HOMEPAGE="https://hpc.github.io/charliecloud/"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+IUSE="buildah ch-grow doc docker examples +pv +squashfs squashfuse"
```

Comment:
You should use `optfeature` from *eutils.eclass* instead for these new optional dependencies. Then you don't have to drop KEYWORDS either, I guess.

Reply:
Indeed, many thanks for this hint! 
I pushed a commit changing this right now (if you prefer, I can squash already now, or of course in the end). Note I still needed to drop `amd64-linux` since newer Python interpreters seem to have dropped gentoo prefix support.